installing/installing_gcp/manually-creating-iam-gcp.adoc

```diff
@@ -5,10 +5,30 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
```

Comment:
These changes have been reviewed in the other PR for 4.7 CCO work, but I copied them over since that one isn't merged yet.